vars/tmsUpload.groovy

```diff
@@ -52,10 +52,15 @@ import static com.sap.piper.Prerequisites.checkScript
      */
     'proxy',
     /**
-     * Toggle to activate a new Golang implementation of the step. Off by default.
+     * DEPRECATED: Toggle to activate a new Golang implementation of the step. The parameter has been deprecated - the new Golang implementation is used now by default.
      * @possibleValues true, false
      */
-    'useGoStep'
+    'useGoStep',
+    /**
+     * Toggle to activate the old Groovy implementation of the step. Off by default. Note that possibility to switch to the old Groovy implementation will be completely removed after February 29th, 2024.
+     * @possibleValues true, false
+     */
+    'useGroovyStep'
```

Comment:
How about setting useGoStep to true if it is not defined? Then there is no need to introduce a new parameter useGroovyStep.

Reply:
Good idea! I will provide respective changes.

Reply:
@o-liver, please take a look. I have provided the change with [b8bfa62](https://github.com/SAP/jenkins-library/pull/4535/commits/b8bfa620260283b43765e596a7d9001a1d745ab6).